src/prefect/cli/work_queue.py

```diff
@@ -365,7 +366,39 @@ async def ls(
     """
     View all work queues.
     """
-    if not pool:
+    if not pool and PREFECT_EXPERIMENTAL_ENABLE_WORK_POOLS.value() is False:
+        table = Table(
+            title="Work Queues",
+            caption="(**) denotes a paused queue",
+            caption_style="red",
+        )
+        table.add_column("Name", style="green", no_wrap=True)
```

Comment:
Is all that follows here just a replication of some old code that was deleted before it should have been ?